### DISCUSSION TITLE
Using Azure SQL server as the feature registry

### DISCUSSION THREAD

**Discussion by bhatsuchi08** [feathr-ai/feathr#623] [Q&A]
Hello all,
We are exploring the possibility of using Feathr as a feature store in our organization and are running POCs. We would like to use SQL server as the feature registry instead of Purview. There is very little code/documentation around this. Could someone please help with configuring the endpoint if I were to use SQL server instead of Purview?

**Comment by xiaoyongzhu** [comment]
Thanks, and the discussion was moved to Slack to make it more interactive.
